config/routes.rb

```diff
@@ -71,6 +71,10 @@
       get '/:id', action: 'show',  as: :petition
     end
 
+    scope '/petitioners', controller: 'petitioners' do
+      get '/:id/verify', action: 'verify', as: :verify_petition_creator
```

Comment:
Wonder whether `verify_petitioner` is a better name for the route to match the path?